SECURITY.md

```diff
@@ -17,6 +17,8 @@ command will deploy the app to Android and iOS, simulator or connected device:
 flutter run --flavor hack
 ```
 
+**NOTE:** this isn't yet support for iOS builds but will soon be.
```

Comment:
do you want to include this file with this PR?

Reply:
Yes - save myself creating another PR. It'll be added to the commit message when I do the squash and merge.